README.md

```diff
@@ -4,29 +4,69 @@ _Bulk import scripts for ingesting large external datasets into OWID's master da
 
-- Scripts that transform it
-- `input/`: an original(ish) copy
-- `output/`: the transformed copy that will be uploaded
-- `standardisation/`: any country name transformations that were needed
+**2. Suggesting chart revisions:** Once a new version of a dataset has been uploaded to the database, the next task is to update the corresponding OWID charts to display the newly available data in place of the old data. Because dataset imports create a _new_ version of an existing dataset rather than overwriting an old version of the same dataset, the relevant charts must be amended to display the new data. The scripts in this repository _suggest_ these chart revisions, which are then manually approved or rejected by an OWID researcher using the [grapher](https://github.com/owid/owid-grapher) admin interface.
 
 ## Development
 
-You should install Python 3.8+ and install required packages:
+1.  install Python 3.8+ and required packages:
 
 ```
 pip install -r requirements.txt
 ```
 
-Historical data transformations might have used Jupyter notebooks, but all recent ones use Python scripts.
+2. Copy `.env.example` to `.env` and change any variables as needed. If you are unsure what to change, ask a member of the OWID data team.
 
-## Updating data
+```
+cp .env.example .env
+```
+
+3. Follow the [setup instructions in the owid-grapher repository](https://github.com/owid/owid-grapher#initial-development-setup) to initialize a local version of the OWID MySQL database.
+
+### Folder structure
+
+Each dataset has its own folder `{institution}_{dataset}/` (e.g. `worldbank_wdi/` for the World Bank World Development Indicators), containing all code and configuration files required to execute the dataset import and suggest the chart revisions.
+
+Typical folder structure:
+
+```
+__init__.py # used for storing dataset constants (e.g. {DATASET_NAME}, {DATASET_VERSION})
+main.py     # executes all 6 steps in sequence
+...         # helper scripts (e.g. `download.py`, `clean.py`)
+input/      # original copy of the dataset
+output/     # the cleaned data to be imported + other *generated* files for steps 1-6
+config/     # *manually* constructed files for steps 1-6.
+```
+
+See [worldbank_wdi/](worldbank_wdi) for a recent example to follow.
+
+## Conventions to follow
+
+Each `{institution}_{dataset}/` folder executes the same 6 steps to import a dataset and suggest chart revisions:
+
+1. Download the data.
+   - Example: [worldbank_wdi/download.py](worldbank_wdi/download.py).
+
+2. Specify which variables in the dataset are to be cleaned and imported into the database.
```

Comment:
That's correct – the variables stored in `variables_to_clean.json` at the end of step 2 are a subset of all variables in the dataset. 

In the case of the WDI bulk import in this PR, I've written `init_variables_to_clean.py` (which constructs `variables_to_clean.json`) so as to only keep the WDI variables that have been used in at least 1 chart. 

My thinking is that this is probably a good rule of thumb for effectively keeping db clutter to a minimum while still providing authors with >90% of the variables they are ever going to use. But more discussion is certainly needed here, and it would be easy enough to alter `init_variables_to_clean.py` to include more variables.